provider/blob/tests/test_versions.rs

```diff
@@ -63,3 +65,92 @@ fn test_v2() {
     let blob_provider = BlobDataProvider::try_new_from_blob(blob.into_boxed_slice()).unwrap();
```

Comment:
Suggestion: use `internal_is_using_v2_bigger_format` and assert that it is false up here in the small blob.